grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -318,6 +320,10 @@ class StockMovementController {
         params.direction = (currentLocation == stockMovement.origin) ? StockMovementType.OUTBOUND :
                 (currentLocation == stockMovement.destination) ? StockMovementType.INBOUND : "ALL"
 
+        if (isCentralPurchasingEnabled) {
+            redirect(controller: 'order', action: "list", params: [orderTypeCode: OrderTypeCode.PURCHASE_ORDER])
```

Comment:
After release when this will go to the develop branch, then this will need to be changed into OrderType